app/pb_kits/playbook/pb_progress_pills/progress_pills.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+module Playbook
+  module PbProgressPills
+    class ProgressPills
+      include Playbook::Props
+
+      partial "pb_progress_pills/progress_pills"
+
+      prop :active, type: Playbook::Props::Number,
+                    default: 0
+      prop :value
+      prop :dark, type: Playbook::Props::Boolean,
+                  default: false
+      prop :steps, type: Playbook::Props::Number,
+                   default: 3
+      prop :title
+
+      def classname
+        generate_classname("pb_progress_pills_kit", dark_class)
+      end
+
+      def dark_class
+        dark ? "dark" : nil
+      end
```

Comment:
Methods that aren't used in the HTML Partial, can be private. An example [here](https://github.com/powerhome/playbook/blob/a3ceed2fdd784ceb5b5d994b71b196407b25311b/app/pb_kits/playbook/pb_body/body.rb#L27)